bundles/org.openhab.binding.miele/src/main/java/org/openhab/binding/miele/internal/handler/MieleBridgeHandler.java

```diff
@@ -324,12 +312,13 @@ private List<HomeDevice> getHomeDevices() throws MieleRpcException {
     }
 
     private Runnable eventListenerRunnable = () -> {
-        String interfaceIpAddress = (String) getConfig().get(INTERFACE);
-        if (!IP_PATTERN.matcher(interfaceIpAddress).matches()) {
-            logger.debug("Invalid IP address for the multicast interface: '{}'", interfaceIpAddress);
+        NetworkInterface networkInterface = getMulticastNetworkInterface(config.ipAddress);
+        if (networkInterface == null) {
             return;
         }
 
+        logger.debug("Using multicast interface {} for gateway {}", networkInterface.getName(), config.ipAddress);
```

Comment:
```suggestion
        logger.debug("Using multicast interface {} ({}) for gateway {}", networkInterface.getName(), networkInterface.getDisplayName(), config.ipAddress);
```

Reply:
> I prefer leaving it as is, since it's only a debug message anyway.

I'm not saying it's a big deal here, but I've met this particular problem many times before, and the point is that `ethernet_32769` is completely meaningless. You won't find this anywhere in Windows, it's something the JVM has "invented". With the device details, you can see which NIC it is.

Reply:
I’m not saying it needs to be adapted, but I wonder how this will turn out for windows users.